web-registry/src/lib/ecocredit.ts

```diff
@@ -213,6 +215,66 @@ export const getTxsByEvent = (msgType: string): Promise<AxiosResponse> => {
   });
 };
 
+export async function getBasket(
+  basketDenom: string,
+): Promise<QueryBasketResponse> {
+  return Promise.resolve({
+    basket: {
+      id: 'basket-1',
+      basketDenom: 'eco.uC.rNCT',
+      name: 'rNCT',
+      disableAutoRetire: false,
+      creditTypeAbbrev: 'C',
+      dateCriteria: {
+        minStartDate: '2022-06-01T00:00:00Z',
+      },
+      exponent: '6',
+    },
+    classes: ['C01', 'C02'],
+  });
+}
+
+export async function getBasketBalances(
+  basketDenom: string,
+): Promise<QueryBasketBalancesResponse> {
+  return Promise.resolve({
+    balances: [
+      {
+        basketId: 'basket-1',
+        batchDenom: 'C01-20170101-20191231-001',
+        balance: '17000',
+      },
+      {
+        basketId: 'basket-1',
+        batchDenom: 'C01-20170101-20191231-002',
+        balance: '23000',
+      },
+      {
+        basketId: 'basket-1',
+        batchDenom: 'C01-20170101-20201231-003',
+        balance: '30000',
+      },
+    ],
+  });
+}
+
+export async function getBatchInfo(
```

Comment:
you could just use the existing `queryEcoBatchInfo` instead of mocking the data

Reply:
trying to remove the BatchInfo mock I've seen myself changing too many things. I'd rather leave it as it is and move on

Reply:
Please have a look at https://github.com/regen-network/regen-web/pull/867

Reply:
super cool! thanks Marie!

Reply:
you can just merge #867 (the target branch being your branch) if that makes sense to you!